.rubocop.yml

```diff
@@ -191,6 +191,12 @@ Style/Next:
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals'
   Enabled: false
 
+# Wakes::ModelConfiguration currently uses method_missing and respond_to_method_missing?
+# but intentonally not does not call super in method_missing.
+Style/MethodMissing:
+  Description: 'This cop checks for the presence of `method_missing` without also defining `respond_to_missing?` and falling back on `super`.'
+  Enabled: false
```

Comment:
@joeosburn-desiringgod did you disable this just because of the one `method_missing` call in `ModelConfiguration`? What would you think about leaving this enabled and selectively disabling it with the magic comment for that method? Or do you disagree with the cop in general?